### DISCUSSION TITLE
Question about "Priority Boosts"

### DISCUSSION THREAD

**Discussion by GflowerG** [winsiderss/systeminformer#975] [Questions / Answers] [answered]
What is the reason for the lack of a function for working with this technology? There are references in the source code, but I do not see the implementation in the application.

**I am writing about it:** https://docs.microsoft.com/en-us/windows/win32/procthread/priority-boosts

**Comment by dmex** [comment, marked as answer]
> What is the reason for the lack of a function for working with this technology?

Nobody requested it, so it wasn't implemented.

> There are references in the source code, but I do not see the implementation

The latest nightly builds includes a new Boost menu for processes and threads 👍
